src/fmu/dataio/dataio.py

```diff
@@ -862,101 +846,39 @@ def __post_init__(self) -> None:
             raise
         logger.info("Ran __post_init__ for InitializeCase")
 
-    def _update_settings(self, newsettings: dict) -> None:
-        """Update instance settings (properties) from other routines."""
-        logger.info("Try new settings %s", newsettings)
+    def _establish_metadata_files(self) -> bool:
+        """Checks if the metadata files and directories are established and creates
+        relevant directories and files if not.
 
-        # derive legal input from dataclass signature
-        annots = getattr(self, "__annotations__", {})
-        legals = {key: val for key, val in annots.items() if not key.startswith("_")}
-
-        for setting, value in newsettings.items():
-            if setting == "restart_from":
-                warn(
-                    "The 'restart_from' argument is deprecated and will be removed in "
-                    "a future version. Please refer to the fmu-dataio documentation "
-                    "for information on how to record information about restart "
-                    "source.",
-                    DeprecationWarning,
-                )
-            if _validate_variable(setting, value, legals):
-                setattr(self, setting, value)
-                logger.info("New setting OK for %s", setting)
-
-    def _establish_pwd_casepath(self) -> None:
-        """Establish state variables pwd and casepath.
-
-        See ExportData's method but this is much simpler (e.g. no RMS context)
+        Returns:
+            False if fmu_case.yml exists (not established), True if it doesn't.
         """
-        self._pwd = Path().absolute()
-
-        if self.rootfolder:
-            self._casepath = Path(self.rootfolder)
-        else:
-            logger.info("Emit UserWarning")
-            warn(
-                "The rootfolder is defaulted, but it is strongly recommended to give "
-                "an explicit rootfolder",
-                UserWarning,
-            )
-            self._casepath = self._pwd.parent.parent
-
-        logger.info("Set PWD (case): %s", str(self._pwd))
-        logger.info("Set rootpath (case): %s", str(self._casepath))
-
-    def _establish_case_metadata(self, force: bool = False) -> bool:
-        if not self._casepath.exists():
-            self._casepath.mkdir(parents=True, exist_ok=True)
+        if not self._metafile.parent.exists():
+            self._metafile.parent.mkdir(parents=True, exist_ok=True)
             logger.info("Created rootpath (case) %s", self._casepath)
-
-        metadata_path = self._casepath / "share/metadata"
-        self._metafile = metadata_path / "fmu_case.yml"
         logger.info("The requested metafile is %s", self._metafile)
-
-        if force:
-            logger.info("Forcing a new metafile")
-
-        if not self._metafile.is_file() or force:
-            metadata_path.mkdir(parents=True, exist_ok=True)
-            return True
-
-        return False
+        return not self._metafile.exists()
```

Comment:
This is a bit confusing, probably indicates a better method name is needed... but hope doc strings clarify